anyway/infographics_utils.py

```diff
@@ -32,10 +34,50 @@
 """
 
 
+@enum.unique
+class WidgetId(enum.Enum):
+    accident_count_by_severity = [1, 'accident_count_by_severity']
+    most_severe_accidents_table = [2, 'most_severe_accidents_table']
+    most_severe_accidents = [3, 'most_severe_accidents']
+    street_view = [4, 'street_view']
+    head_on_collisions_comparison = [5, "head_on_collisions_comparison"]
+    accident_count_by_accident_type = [6, "accident_count_by_accident_type"]
+    accidents_heat_map = [7, "accidents_heat_map"]
+    accident_count_by_accident_year = [8, "accident_count_by_accident_year"]
+    injured_count_by_accident_year = [9, "injured_count_by_accident_year"]
+    accident_count_by_day_night = [10, "accident_count_by_day_night"]
+    accidents_count_by_hour = [11, "accidents_count_by_hour"]
+    accident_count_by_road_light = [12, "accident_count_by_road_light"]
+    top_road_segments_accidents_per_km = [13, "top_road_segments_accidents_per_km"]
+    injured_count_per_age_group = [14, "injured_count_per_age_group"]
+    vision_zero = [15, "vision_zero"]
+    accident_count_by_driver_type = [16, "accident_count_by_driver_type"]
+    accident_count_by_car_type = [17, "accident_count_by_car_type"]
+    injured_accidents_with_pedestrians = [18, "injured_accidents_with_pedestrians"]
+    accident_severity_by_cross_location = [19, "accident_severity_by_cross_location"]
+    motorcycle_accidents_vs_all_accidents = \
+        [20, "motorcycle_accidents_vs_all_accidents"]
+    accidents_count_pedestrians_per_vehicle_street_vs_all = \
+        [21, "accidents_count_pedestrians_per_vehicle_street_vs_all"]
+
+    def get_rank(self) -> int:
+        return self.value[0]
+
+    def get_name(self) -> str:
+        return self.value[1]
+
+
 class Widget:
-    def __init__(self, name, rank, items, text=None, meta=None):
-        self.name = name
-        self.rank = rank
+    # def __init__(self, name, rank, items, text=None, meta=None):
+    #     self.name = name
+    #     self.rank = rank
+    #     self.items = items
+    #     self.text = text
+    #     self.meta = meta
+    #
```

Comment:
remove

Reply:
Sure, this is WIP. I kept the old code for easy reference.